addons/l10n_account_edi_ubl_cii_tests/tests/test_xml_cii_fr.py

```diff
@@ -288,6 +288,7 @@ def test_encoding_in_attachment_facturx(self):
 
     def test_import_partner_facturx(self):
         self._test_import_partner('facturx_1_0_05', 'factur-x.xml')
+        self._test_import_and_create_partner('facturx_1_0_05')
```

Comment:
I don't understand why you don't unlink the partner, then call the import, then check the import is well created instead of this weird helper method doing something very specific... :thinking: